cmake/kokkos_tribits.cmake

```diff
@@ -204,10 +204,15 @@ ENDMACRO()
 
 FUNCTION(KOKKOS_SET_LIBRARY_PROPERTIES LIBRARY_NAME)
```

Comment:
Are the changes in this file used anywhere? I don't find any call to it that changed.